hicommonwealth/commonwealth
# Added community name search for stake transactions history

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6675

## Description of Changes
- Added community name/symbol search in stake transactions history tab
## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_MY_COMMUNITY_STAKE_PAGE_ENABLED` in `.env`
- Visit `/myCommunityStake` -> "Transaction History" tab
- Verify you can search communities with name and symbol

## Deployment Plan
N/A

## Other Considerations
N/A
